lineplot/index.html

```diff
@@ -1,126 +1,86 @@
-<html ng-app="lineplotApp">
+<html id="lineplot">
 
 <head>
   <script src='/chaise/chaise-config.js'></script>
-  <script src='/chaise/scripts/vendor/jquery-1.11.1.min.js'></script>
+  <script src='/chaise/scripts/vendor/jquery-3.4.1.min.js'></script>
```

Comment:
When I pull your changes and upload the code to my directory on prisms, I get an error that this file is not available. This current path assumes the file will be fetched from the root of the server instead of relative to the user's current location. 

For now, can you change the script tags in this html document to point to chaise relative to your current location? i.e. `../chaise/scripts/vendor/...` and `../chaise/common/...` for links further down the page. More info about why this is bad can be seen in issue #67 .

Reply:
Updated it with relative paths.